eclass/toolchain.eclass

```diff
@@ -1816,48 +1864,68 @@ gcc_do_make() {
 #---->> src_test <<----
 
 toolchain_src_test() {
-	cd "${WORKDIR}"/build || die
+	# GCC's testsuite is a special case.
+	#
+	# * Generally, people work off comparisons rather than a full set of
+	#   passing tests.
+	#
+	# * The guality tests are for debug info quality and are especially
```

Comment:
```guality``` ;)

Reply:
Not a typo! They're the debug quality tests. I'll clarify it though.